tokenizer/tokenizer.py

```diff
@@ -29,3 +30,10 @@ def tokenize_source(source, t_config):
     return f(source, t_config)
 
 
+def watermarker(submission_text):
+    watermarks = []
+    mark = re.findall('[\u200c\u200b]+', submission_text)
+    for i in mark:
+        watermarks.append(int(i.replace('\u200b','1').replace('\u200c','0'),2))
+    if watermarks:
+        return(watermarks)
```

Comment:
It would probably make sense to always return a list instead of changing the empty list to the implicit `None` return value.